FinSearchUnified/Tests/Subscriber/WidgetsTest.php

```diff
@@ -11,12 +11,12 @@
 
 class WidgetsTest extends TestCase
 {
-    protected static $ensureLoadedPlugins = [
-        'FinSearchUnified' => [
-            'ShopKey' => '0000000000000000ZZZZZZZZZZZZZZZZ',
-            'ActivateFindologic' => true
-        ],
-    ];
+    public static function setUpBeforeClass()
```

Comment:
Why was this change necessary? `$ensureLoadedPlugins` should already have worked (at least theoretically).

Reply:
Since we have disabled Findologic by default in the last version, it did not get activated by using `ensureLoadedPlugins`'s value after a fresh setup of the environment. Not sure why though.

Reply:
Well, that's unfortunate.